app/styles/components/filter-sidebar.scss

```diff
@@ -0,0 +1,47 @@
+.filter-sidebar-group {
+  display: flex;
+  flex-direction: column;
+}
+
+.filter-sidebar-group--expanded .filter-sidebar-group-label .fa {
```

Comment:
Be aware that you made the css selector less accurate while deleting `&.` in front of it. The exact replacement would have been:
```scss
.filter-sidebar-group .filter-sidebar-group--expanded .filter-sidebar-group-label .fa
``` 

or indent the those rules in the parenting block `.filter-sidebar-group { }`